ORBIT/phases/install/install_phase.py

```diff
@@ -33,8 +34,68 @@ def __init__(self, weather, **kwargs):
 
         self.extract_phase_kwargs(**kwargs)
         self.initialize_environment(weather, **kwargs)
+
+        self.processes = self.build_processes(processes)
         self.availability = kwargs.get("availability", None)
 
+    def build_processes(self, input):
+        """
+
+        Parameters
+        ----------
+        input :
+        """
+
+        processes = {**default_processes, **input}
```

Comment:
A user can now pass a dictionary containing modified processes into an InstallPhase (and eventually ProjectManager, just haven't wired that up yet) to override any of the default process definitions.